docs/tutorials/querying_with_clinical_trials.md

```diff
@@ -46,7 +46,7 @@ You can see the in-line citations for each clinical trial used as a response for
 to see more data on the specific contexts that were used to answer the query:
 
 ```python
-print(answer_response.session.contexts)
```

Comment:
This doesn't seem right - this is PQASession right? Not aiohttp session?

Reply:
Yeah nice catch, thanks. Just fixed this